src/components/autocompleteInput.js

```diff
@@ -579,23 +580,23 @@
     /*
      * Convert `value` state into something the `value` prop of the `Autocomplete` component will accept with the right settings
      */
-    const getValue = () => {
+    const getValue = (testVAlue = value) => {
       if (isListProperty) {
-        return value;
+        return testVAlue;
```

Comment:
small typo

Reply:
+ maybe rename this to a better variable name

Reply:
This should be fixed in the next commit